Code/GraphMol/ChemReactions/Reaction.h

```diff
@@ -140,7 +140,7 @@ class RDKIT_CHEMREACTIONS_EXPORT ChemicalReaction : public RDProps {
       ROMol *agent = new ROMol(**iter);
       m_agentTemplates.push_back(ROMOL_SPTR(agent));
     }
-    dp_props = other.dp_props;
+    d_props = other.d_props;
```

Comment:
Out of curiosity, why the change from dp_props to d_props?

Reply:
because it's not a pointer. At least I don't think it is

Reply:
Correct :)